### PR TITLE
Doc: Remove include statements for screenshots not rendering properly

### PR DESCRIPTION
**Pull request by karenzone**
Fixes: #15882 

Screenshots are not rendering properly inside widgets. The widgets are documenting guided flows, and the screenshots are not critical for understanding.  Note that I am not deleting the image files. They're still in place for when/if we decide to find a workaround and reimplement them. 

**PREVIEW:** https://logstash_bk_15981.docs-preview.app.elstc.co/guide/en/logstash/master/dashboard-monitoring-with-elastic-agent.html

### CODE REVIEW

File: docs/static/tab-widgets/install-agent.asciidoc [hunk 1]
```diff
@@ -20,20 +15,11 @@ It takes about a minute for {agent} to enroll in {fleet}, download the
...
 // tag::standalone[]
-. When the **Add Agent flyout** appears, navigate to the **Run standalone** tab
-+
---
-[role="screenshot"]
-image::../monitoring/images/integration-agent-add-standalone.png[Add agent flyout in {kib}]
---
-. Configure the agent. Follow all the instructions in **Install Elastic Agent on your host**
+. When the **Add Agent flyout** appears, navigate to the **Run standalone** tab.
+. Configure the agent. Follow all the instructions in **Install Elastic Agent on your host**.
```
Comment (karenzone) [inline docs/static/tab-widgets/install-agent.asciidoc:22]:
```suggestion
. Configure the agent. Follow the instructions in **Install Elastic Agent on your host**.
```

**Comment by karenzone** [comment]
run docs-build

**Review by robbavey — APPROVED** [review]
lgtm

File: docs/static/tab-widgets/install-agent.asciidoc [hunk 2]
```diff
@@ -1,10 +1,5 @@
 // tag::fleet-managed[]
-. When the **Add Agent flyout** appears, stay on the **Enroll in fleet** tab
-+
---
-[role="screenshot"]
-image::../monitoring/images/integration-agent-add.png[Add agent flyout in {kib}]
---
```
Comment (karenzone) [inline docs/static/tab-widgets/install-agent.asciidoc:7]: @karenzone note to self: Note that you removed references, but did not delete image files from the repo

**Comment by karenzone** [comment]
@logstashmachine backport 8.13

**Comment by karenzone** [comment]
6918

**Comment by karenzone** [comment]
@logstashmachine backport 8.14

**Pull request merged**